folium/features.py

```diff
@@ -197,8 +199,46 @@ def __init__(self, data):
             {% endmacro %}
             """)
 
+class TopoJson(MacroElement):
+    def __init__(self, data, object_path):
+        """TODO docstring here.
+        """
+        super(TopoJson, self).__init__()
+        self._name = 'TopoJson'
+        if 'read' in dir(data):
+            self.data = data.read()
+        elif type(data) is dict:
+            self.data = json.dumps(data)
+        else:
+            self.data = data
+
+        self.object_path = object_path
+
+        self._template = Template(u"""
+            {% macro script(this, kwargs) %}
+                var {{this.get_name()}}_data = {{this.data}};
+                var {{this.get_name()}} = L.geoJson(topojson.feature(
+                    {{this.get_name()}}_data,
+                    {{this.get_name()}}_data.{{this.object_path}}
+                    )).addTo({{this._parent.get_name()}});
+            {% endmacro %}
+            """)
+    def render(self,**kwargs):
+        super(TopoJson,self).render(**kwargs)
+
+        figure = self.get_root()
+        assert isinstance(figure,Figure), ("You cannot render this Element "
+            "if it's not in a Figure.")
+
+        figure.header.add_children(\
```

Comment:
I guess you're right. It's a (bad?) habbit I took not to rely on brackets for multi-line.
Tell me and I'll put them away.